### DISCUSSION TITLE
Signature declarations can only be used in TypeScript files is error in Step 3

### DISCUSSION THREAD

**Discussion by colleenpridemore** [skills/.github#653] [Code with Copilot]
![Screenshot 2024-09-24 165050](https://github.com/user-attachments/assets/694ad99b-467b-46ca-9ac2-3098e3709541)
why am I getting this error?  I don't find anyone else getting this error.

**Comment by brunomsdev** [comment]
You probable got this error because you didn't add curly braces after it. I got the same error once and it was that.

**Comment by Neuwair** [comment]
To fix this error you need to include curly braces, try again.

**Comment by Lerenah3** [comment]
Dead bot.
